Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/BoundsControl/Visuals/Links.cs

```diff
@@ -124,11 +124,15 @@ internal void UpdateLinkScales(Vector3 currentBoundsExtents)
 
         internal void Flatten(ref int[] flattenedHandles)
         {
-            if (flattenedHandles != null && linkRenderers != null)
+            if (flattenedHandles != null)
             {
                 for (int i = 0; i < flattenedHandles.Length; ++i)
                 {
-                    linkRenderers[flattenedHandles[i]].enabled = false;
+                    Renderer linkRenderer = links[flattenedHandles[i]].transform.gameObject.GetComponent<Renderer>();
+                    if (linkRenderer)
```

Comment:
Nit: Be consistent with != null or (linkRenderer) with the lines above (i.e. above we check for != null, here we do the implicit null check)